colossalai/utils/profiler/common.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+from .comm_profiler import CommProfiler
+from typing import List
+from torch.utils.tensorboard import SummaryWriter
+
+
+class ProfilerEvent(Enum):
+    '''
+    Event types to profile
+    '''
+    COMM = CommProfiler

```

Comment:
Do not use abbreviation for API exposed to the user.

Reply:
It is not a common practice to put a class as the enum value.  It seems not standard to me even though it works. What do you think @feifeibear .

Reply:
Also weird to me

Reply:
@kurisusnowdeng Any idea about this part?

Reply:
We can also use some keys to redirect to the profiler classes in a dict. But, I think this is a redundant step.

Reply:
Actually, using Enum as an option is quite common in C++ and Rust, but I have no idea whether this is appropriate for Python.